core/domain/exp_jobs_one_off.py

```diff
@@ -491,6 +491,9 @@ def map(item):
         # invalid parent-child relations that we find.
         err_dict = {}
 
+        # To store all the invalid html strings.
```

Comment:
Minor nit: make this a complete sentence.

Reply:
Done!